frontend/src/routes/Components/TertiaryButton.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from "react";
+class TertiaryButton extends Component {
+  render() {
+    const tertiary = {
+      backgroundColor: "white",
+      borderRadius: "0.625rem",
+      border: "0.05rem #949494 solid",
+      color: "#949494",
+      minWidth: "12rem",
```

Comment:
i don't understand why this button's code is not similar to the other buttons.. how come it has a min width but the others don't? + also what about padding/text-align center?

Reply:
**Added minWidth to other button elements**